app-info/eos-extra.xml

```diff
@@ -456,4 +456,1432 @@
       <value key="GnomeSoftware::popular-background">https://d3lapyynmdp1i9.cloudfront.net/thumbnails/yelp/yelp-thumb.jpg</value>
     </metadata>
   </component>
+  <component type="desktop" merge="append">
+    <id>com.endlessm.all_that_is_interesting.en</id>
+    <kudos>
+      <kudo>GnomeSoftware::popular</kudo>
+    </kudos>
+    <bundle type="flatpak"></bundle>
```

Comment:
Is it OK to leave the `<bundle>` element empty? Shouldn’t we be specifying the flatpak’s app ID as the `<bundle>` content?

Reply:
Yes it's OK now. Since we revert joaquim patch, all popular apps are marked as wildcards, and _refine_wilcard in flatpak-plugin finds the real app for this particular wildcard. The real app already has the correct flatpak app-id as source so we don't need to set this in the app.

We although need the <bundle> to make flatpak plugin adopt this wildcard app.

Reply:
OK.